### PR TITLE
Fix express routing of static file

### PR DESCRIPTION
**Pull request by drewbutlerbb4**
Resolves: #159

Fixes bug when pages are reloaded, but can not load correctly. Also adds a 404 page for any route in mlx which does not match with another specified route.

Signed-off-by: Andrew-Butler <Andrew.Butler@ibm.com>

### CODE REVIEW

File: dashboard/origin-mlx/src/pages/MetaDetailPage.tsx [hunk 1]
```diff
@@ -37,33 +37,50 @@ function MetaDetailPage(props: MetaDetailPageProps) {
...
 
   const { store } = useContext(StoreContext)
   const { api } = store.settings.endpoints
   const API = api.value || api.default
 
+  const fetchTemplates = () => {
+    fetchAssetTemplates(API, type, asset)
+    .then((template: any) => {
+      if (props.asset.search) {
+        const url_params = props.asset.search.substring(1).split("&")
```
Comment (ckadner) [inline dashboard/origin-mlx/src/pages/MetaDetailPage.tsx:50]: is there some escaping that needs to be taken into consideration when splitting search strings? i.e. could the value of a parameter include an escaped `&` character or escaped `=` sign that would throw off the logic here?

**Review by ckadner — COMMENTED** [review]
Thanks @drewbutlerbb4 -- I like adding a 404 page 👍🏻 

Just a quick question on the parameter splitting

**Comment by ckadner** [comment]
@yhwang @Tomcli -- if you have no objections, we should merge this

**Review by ckadner — APPROVED** [review]

**Review by yhwang — APPROVED** [review]
/lgtm

**Pull request merged**